### ISSUE TITLE
regex error for video (akamaized pattern)

### ISSUE THREAD

**Issue by partriv** [pytube/pytube#336]
I am getting this error 
<class 'pytube.exceptions.RegexMatchError'>, 

RegexMatchError('regex pattern (yt\\.akamaized\\.net/\\)\\s*\\|\\|\\s*.*?\\s*c\\s*&&\\s*d\\.set\\([^,]+\\s*,\\s*(?P<sig>[a-zA-Z0-9$]+)\\() had zero matches',)

When attempting to construct the YouTube instance for this video https://youtu.be/IABRgZH12YA

I've also tried the long link (https://www.youtube.com/watch?v=IABRgZH12YA&feature=youtu.be) and it did not work.

repro:
```
from pytube import YouTube
yt = YouTube("https://youtu.be/IABRgZH12YA")
```

pytube version 9.3.5

**Comment by rafagarciac** [comment]
Same error ! Check out this code:
```
from pytube import YouTube as Youtube

yt = Youtube('https://www.youtube.com/watch?v=nsm4ReJaED0')
yt.streams.all() 
```

Exception:
> **_pytube.exceptions.RegexMatchError: regex pattern (yt\.akamaized\.net/\)\s*\|\|\s*.*?\s*c\s*&&\s*d\.set\([^,]+\s*,\s*(?P<sig>[a-zA-Z0-9$]+)\() had zero matches_**

**Comment by partriv** [comment]
seems like the fix mentioned in [this thread](https://github.com/nficano/pytube/issues/333#issuecomment-436668766)  applies to this problem as well. When I update the regex pattern in cipher.py, it works, but is that actually the right fix?

**Comment by partriv** [comment]
dupe of issue #333, closing this. For now, use the fix as detailed in #333
